src/components/street-geo.js

```diff
@@ -76,13 +92,46 @@ AFRAME.registerComponent('street-geo', {
       }
     }
 
-    // if state is not clipping, then disable it
     if (this.google3d) {
+      // Handle clipping updates
       if (data.enableClipping) {
         this.google3d.setAttribute('obb-clipping', '');
       } else {
         this.google3d.removeAttribute('obb-clipping');
       }
+
+      // Handle blending updates
+      if (data.blendingEnabled) {
+        if (data.blendMode) {
+          this.google3d.setAttribute(
+            'blending-opacity',
+            this.returnBlendMode(data.blendMode)
+          );
+        }
+      } else {
+        this.google3d.removeAttribute('blending-opacity');
+        if (oldData.blendingEnabled) {
+          // If blending was previously enabled and now disabled, recreate the tiles
+          const currentEl = this.google3d;
+          this.el.removeChild(currentEl);
+          this.google3d = null;
+          this.google3dCreate();
+        }
+      }
+
+      // regenerate tiles if blending enabled in editor
+      const dataDiff = AFRAME.utils.diff(oldData, data);
```

Comment:
You already have `updatedData` at the beginning of `update()`, you can reuse it.